rpcserver.go

```diff
@@ -3267,6 +3267,10 @@ func (r *rpcServer) fetchPendingOpenChannels() (pendingOpenChannels, error) {
 		utx := btcutil.NewTx(localCommitment.CommitTx)
 		commitBaseWeight := blockchain.GetTransactionWeight(utx)
 		commitWeight := commitBaseWeight + input.WitnessCommitmentTxWeight
+		confirmationHeight := uint32(0)
```

Comment:
nit: declare as `var confirmationHeight uint32`? And add empty line above to visually group this a bit.